core-web/libs/sdk/editor/jest.config.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+export default {
+    displayName: 'sdk-editor',
```

Comment:
this should be `@dotcms/editor` right?
cc: @fmontes

Reply:
The package name is @dotcms/editor, but in other places is called sdk-editor to get same naming as sdk-react/sdk-client, like test, build, etc

Reply:
the name in the package.json needs to be @dotcms/editor in all the other places, like the project `sdk-editor`

Reply:
`sdk-editor` is nx project name
`@dotcms/editor` is the npm package name @zJaaal